# This advisory is derived from https://github.com/victims/victims-cve-db which
# is licensed under CC-BY-SA 4.0
---
identifier: "CVE-2017-18342"
package_slug: "pypi/PyYAML"
title: "Improper Input Validation"
description: "In PyYAML, the `yaml.load()` API could execute arbitrary code if used
  with untrusted data."
date: "2019-06-24"
pubdate: "2018-06-27"
affected_range: "<5.1"
fixed_versions:
- "5.1"
affected_versions: "All versions before 5.1"
not_impacted: "All versions starting from 5.1"
solution: "Upgrade to version 5.1 or above."
urls:
- "https://github.com/yaml/pyyaml/pull/74"
- "https://github.com/yaml/pyyaml/blob/master/CHANGES"
cvss_v2: "AV:N/AC:L/Au:N/C:P/I:P/A:P"
cvss_v3: "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
uuid: "9f23df3d-089c-4b26-b36c-844f8abb87cf"
